tutorials/gas-price.md

```diff
@@ -0,0 +1,26 @@
+# How to configure gas price
+
+In most cases the gas price can be left to the defaults, which ensures that the price
+is greater than the minimum gas price accepted by the core node.
+
+The gas price can also be configured manually with the flag `--rollkit.da_gas_price`:
+
+```bash
+gmd start --rollkit.da_gas_price=0.1 [existing flags...] // [!code --]
```

Comment:
`[!code --]` will show this line as if it is being removed, FYI. may not be necessary. opposite, `[!code ++]` will show it being added